cluster/gce/manifests/kube-controller-manager.manifest

```diff
@@ -13,12 +13,24 @@
   }
 },
 "spec":{
+"securityContext": {
+  "runAsUser": {{runAsUser}},
+  "runAsGroup": {{runAsGroup}}
```

Comment:
It's a good idea to set `runAsNonRoot: true` (mostly for easy policy checking), but it would conflict with the sed commands here.

Reply:
I agree with @tallclair - we should do another pass after to add `runAsNonRoot: true` to all components.

Reply:
Sounds good will do in a followup for all the oss master static pods.